joint_limits/CMakeLists.txt

```
@@ -73,13 +74,14 @@ if(BUILD_TESTING)
   find_package(rclcpp REQUIRED)
 
   ament_add_gmock(joint_limits_rosparam_test test/joint_limits_rosparam_test.cpp)
-  target_link_libraries(joint_limits_rosparam_test joint_limits)
+  target_link_libraries(joint_limits_rosparam_test joint_limits fmt::fmt)
   target_compile_definitions(
     joint_limits_rosparam_test
-    PRIVATE PARAMETERS_FILE_PATH="${CMAKE_CURRENT_LIST_DIR}/test/")
+    PRIVATE PARAMETERS_FILE_PATH="${CMAKE_CURRENT_LIST_DIR}/test/"
+  )
```

```suggestion
    PRIVATE PARAMETERS_FILE_PATH="${CMAKE_CURRENT_LIST_DIR}/test/")
```
Any particular reason for this change?

---

Nope, I have reverted it